TwitchDownloaderCore/Extensions/M3U8Extensions.cs

```diff
@@ -124,7 +124,9 @@ public static string GetResolutionFramerateString(this M3U8.Stream stream)
         /// </summary>
         public static M3U8.Stream BestQualityStream(this M3U8 m3u8)
         {
-            var source = Array.Find(m3u8.Streams, x => x.MediaInfo.Name.Equals("source", StringComparison.OrdinalIgnoreCase));
+            var source = Array.Find(
+                m3u8.Streams, x => x.MediaInfo.Name.Equals("source", StringComparison.OrdinalIgnoreCase) ||
```

Comment:
```suggestion
                m3u8.Streams, x => x.MediaInfo.Name.Contains("source", StringComparison.OrdinalIgnoreCase) ||
```

Sometimes source is in braces for live M3U8 responses.
https://github.com/lay295/TwitchDownloader/blob/e90a07d707d3a011e16bfc4ee71db666134c4128/TwitchDownloaderCore.Tests/M3U8ExtensionTests.cs#L24-L40
I know we don't support recording streams live, but maybe we will in the future or Twitch will change the VOD M3U8 response again.

Reply:
I missed that in your original comment above, thanks!